bundle-size/README.md

```diff
@@ -0,0 +1,79 @@
+# Bundle size action
+
+This GitHub Action compares two webpack compilation stats files and comments on PRs with a description of the difference.
+
+## Features
+
+- Compares the build stats of main against build stats of the PR
+- Reports the result in markdown tables in a PR comment
+
+## Usage
+
+- Your CI that runs when merging main _must_ create and upload a `main-branch-stats` asset (see [troubleshooting](#troubleshooting) for more info).
+- Add a workflow to your Github repository as in the example below.
+
+## Workflow example
+
+```yaml
+name: Bundle Stats
+
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+  actions: read
+
+jobs:
+  compare:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - uses: grafana/plugin-actions/bundle-size@main
+```
+
+## Options
+
+The following options can be passed to this action:
+
+- `token`: A github token with write access to pull requests and content (defaults to `github.token`).
+- `node-version`: The version of node to use (defaults to `20`).
+- `threshold`: The minimum entrypoints total percentage difference required to post comments in PRs (defaults to `5`).
+
+## Troubleshooting
+
+**How do I add the `main-branch-stats` asset to my github repo?**
+
+To add and update `main-branch-stats` in your github repo you can make the following changes to your workflow that runs on merges to your default (`main`) branch:
+
+1. Replace the frontend build command so on pushes to main the stats.json file is generated by webpack:
+
+```diff
```

Comment:
which file is this? maybe let's use the one we use in our create-plugin template as example?

Reply:
Have updated the readme to mention file paths with the assumption that we're using create-plugin workflows.